### PR TITLE
New wallet wizard redesign

### PR DESCRIPTION
**Pull request by bauerj**
This implements the new design presented in #3916.

Also:
 * `HistoryList`'s `format_date` was moved to util
 * New icon for "Create new file"
 * Icon for "Open file" was changed to something more modern

Before:
python_2018-03-07_22-41-12
After:

Moving image

Closes: #3916
Closes: #3914

### CODE REVIEW

**Comment by bauerj** [comment]
Hey,

I couldn't reproduce the width issue on my device but I think it should be better now. Can you test again?

**Comment by SomberNight** [comment]
My previous remarks have been addressed; thanks. New ones here :P

This is how it looks for me now:
pr_4069_take2_pic2

If I change the min window height to 500 as in `InstallWizard.setMinimumSize(600, 500)`, then it becomes:
pr_4069_take2_pic1
(maybe that's too much, I don't know, but there should not be a vertical scrollbar in the outer box)

-----

The text in the fields of the "recent wallets" table should not be editable.

**Comment by bauerj** [comment]
Addressed again 😎

**Comment by SomberNight** [comment]
Thanks.
As for functionality, I like it. :)
However, I don't really like the fact that at this point `run_and_get_wallet` is ~240 lines... >.< Creating the layout is about 90 lines.. do you think it would be possible to move the layout-creating part into a separate method?

**Comment by bauerj** [comment]
Sure, I'll refactor it next week.

File: gui/qt/installwizard.py [hunk 1]
```diff
@@ -148,38 +146,116 @@ def __init__(self, config, app, plugins, storage):
...
+        palette.setColor(QPalette.Disabled, QPalette.Text, palette.color(QPalette.Normal, QPalette.Text))
+        self.wallet_name.setPalette(palette)
+        self.wallet_opened = QLabel(_('Last opened:'))
+
+        layout_wallet_info = QHBoxLayout()
+        layout_wallet_info.addWidget(self.wallet_name)
+        layout_wallet_info.addWidget(self.wallet_opened)
+        layout_wallet.addLayout(layout_wallet_info)
+
+        self.wallet_message = QLabel('Enter your password or choose another wallet.')
```
Comment (SomberNight) [inline gui/qt/installwizard.py]: translate string _()

**Comment by SomberNight** [comment]
Maybe the name of the selected wallet should always stay a QLineEdit.
Currently, the changing between input field/static text, has weird consequences.
If you have a wallet named `wallet6`, and would want to create a wallet named `wallet61`, the field will change from an input field to static text after typing `wallet6`.
Further, if there is some existing string in the input field, and you try to modify it in the middle, after every keypress (textEdited callback I guess), the cursor will be placed at the end.

**Comment by bauerj** [comment]
I agree, I didn't think about wallet names that start with the name of an existing wallet. Both should be fixed now.

**Pull request closed without merging**